shell/browser/ui/views/electron_constrained_window_views_client.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ELECTRON_BROWSER_UI_VIEWS_CHROME_CONSTRAINED_WINDOW_VIEWS_CLIENT_H_
+#define ELECTRON_BROWSER_UI_VIEWS_CHROME_CONSTRAINED_WINDOW_VIEWS_CLIENT_H_
```

Comment:
The `#ifndef` name should be the same as the path, so in this case it would be `SHELL_BROWSER_UI_VIEWS_ELECTRON_CONSTRAINED_WINDOW_VIEWS_CLIENT_H_`.